package.json

```diff
@@ -15,6 +15,7 @@
     "transpile": "cross-env BABEL_DISABLE_CACHE=1 babel src --out-dir lib",
     "start": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot",
     "prepublishOnly": "npm run build && npm run transpile",
+    "preinstall": "node ./config/preinstall.js",
```

Comment:
Why is this done in `preinstall`? Shouldn't it be in `prepare`? (IIUC, `preinstall` will also run when installing iTowns as a dependency of the project you're currently developing in, not only when hacking on iTowns proper).

Reply:
Yes, thanks, I didn't choose the correct script. By cons I would like to warn before the **`npm install`** (the download of Chromium by example) I should use **`prepublish`**.

Reply:
From https://docs.npmjs.com/misc/scripts#prepublish-and-prepare, `prepare` is the same as `prepublish`, and should probably be preferred here. It will _also_ run before publishing, but :man_shrugging:

Reply:
Ok, I moved the code in **`prepare`** script.